src/Bot.ts

```diff
@@ -170,4 +175,19 @@ export class Bot extends builder.UniversalBot {
             session.send(Strings.remove_like_message);
         }
     }
+
+    // method for handling incoming payloads from popup signin
+    private async verifySigninState (event: builder.IEvent, query: teams.ISigninStateVerificationQuery, callback: (err: Error, body: any, status?: number) => void): Promise<void>
+    {
+        let session = await loadSessionAsync(this, event);
+
+        if (session)
+        {
+            let magicNumber = query.state;
+
+            session.clearDialogStack();
+            session.send(session.gettext(Strings.popupsignin_successful) + magicNumber);
```

Comment:
Add the %s placeholder to popupsigin_successful. Change this to
session.send(Strings.popupsignin_successful, magicNumber)

Reply:
Done.